plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/configuration/DatabaseMetadata.java

```diff
@@ -38,20 +38,35 @@ public int getVersion() {
     return version;
   }
 
-  public static DatabaseMetadata fromDirectory(final Path databaseDir) throws IOException {
-    final File metadataFile = getDefaultMetadataFile(databaseDir);
+  public static DatabaseMetadata lookUpFrom(final Path databaseDir, final Path dataDir)
+      throws IOException {
+    File metadataFile = getDefaultMetadataFile(dataDir);
+    final boolean shouldLookupInDatabaseDir = !metadataFile.exists();
```

Comment:
I think we should log a WARN or INFO if metadata exists at both locations.  

In a future revision (1.4?) we should consider deleting the metadata from the old location if it matches what is in the new locaiton.